src/gui/search/searchwidget.cpp

```diff
@@ -242,12 +241,18 @@ QTabWidget *SearchWidget::searchTabs() const
 void SearchWidget::on_searchButton_clicked()
 {
     if (Utils::Misc::pythonVersion() < 0) {
-        m_mainWindow->showNotificationBaloon(tr("Search Engine"), tr("Please install Python to use the Search Engine."));
+        // unconditional notifications, user can not disable it
+        Notifications::Request()
+            .title(tr("Search Engine"))
+            .message(tr("Please install Python to use the Search Engine."))
+//          .setWidget(this)
+            .severity(Notifications::Severity::Error)
+            .exec();
```

Comment:
I forgot to add that there should also be some signal connected to the Notifications Manager.